azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/utilities/BottomCellViewHolder.kt

```diff
@@ -4,89 +4,20 @@
 package com.azure.android.communication.ui.calling.utilities
 
 import android.view.View
-import android.widget.ImageView
 import android.widget.TextView
-import androidx.core.content.ContextCompat
-import androidx.core.view.AccessibilityDelegateCompat
-import androidx.core.view.ViewCompat
-import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
 import androidx.recyclerview.widget.RecyclerView
 import com.azure.android.communication.ui.R
-import com.microsoft.fluentui.persona.AvatarView
 
-internal class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val imageView: ImageView = itemView.findViewById(R.id.cell_icon)
-    private val avatarView: AvatarView =
-        itemView.findViewById(R.id.azure_communication_ui_participant_list_avatar)
+internal open class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
Rename class to `BottomCellTitleViewHolder` for better clarity

Reply:
@phileo99  If we only have one cellviewholder, I think we can keep the name bottomcellviewholder for both item and tittle.

Reply:
Included in 0beaa7d - I've changed the class name to BottomCellTitleViewHolder, for now. I can change it back to the original if we want it that way.